test/integration/inputs/happypath/happypath.yaml

```diff
@@ -13,7 +13,7 @@ helmDefaults:
 releases:
```

Comment:
Can we probably place this at `integration/inputs/happypath.yaml` as sinilar as other helmfile.yaml files like `integration/inputs/issue.1857.yaml` for consistency?

Reply:
`happypath.yaml` depends `environment.values.yaml` and `value.yaml`. I put the related files in a directory, updated clearly. so they all in `test/integration/inputs/happypath` dir.

Reply:
@mumoshu

Reply:
The current changes unify the structure, and the remaining optimizations require other PRS, reducing the burden of code review.

Reply:
Why only happypath-related files are isolated in their own directory, even though other helmfile.yaml files like `integration/inputs/issue.1857.yaml` are in a common, upper-layer directory?

Reply:
ok, I will isolate each state into a separate directory. WDYT? @mumoshu

Reply:
I think either of the followings seem to make sense to me:

- Put every input file (including helmfile.yaml and the values.yaml files) into a common `inputs` directory and put every output file into a common `outputs` directory
- Put input files to a `test/integration/cases/$TEST_CASE_NAME/input/` directory and the snapshot file to `test/integration/cases/$TEST_CASE/output` directory, so that you can easily pair the input vs the output within the same `$TEST_CASE_NAME` directory

I slightly prefer the latter as it's more extensible and easy to group related files up.

The current state of this pull request seems to be floating somewhere between the two and that might be why it doesn't look very consistent to me. Please correct me if I'm missing something though.

Reply:
make sense. I will update code.